pkg/netrc/netrc_test.go

```diff
@@ -233,10 +233,14 @@ var hasNetrcFileDirNetrcContent string
 
 func TestLoad(t *testing.T) {
 	currentDir, err := os.Getwd()
+	netrcFileName := getNetrcFileName()
+
 	if err != nil {
 		t.Fatalf("os.Getwd() = %v", err)
 	}
-	testDataDir := path.Join(currentDir, "testdata", "load_test")
+	testDataDir := filepath.Join(currentDir, "testdata", "load_test")
```

Comment:
Can we change `load_test` to `netrc_test`,  `load_netrc_test` or anything similar? Not that it is a big deal, but "load test" usually refers to performance tests under heavy request loads, and I think we should avoid this potential confusion if possible

Reply:
I suggest this should be part of another PR. In this PR I only wanted to address the Windows-specific issues.